recipes/botan/all/conanfile.py

```diff
@@ -35,6 +49,20 @@ class BotanConan(ConanFile):
                        'with_sqlite3': False,
                        'with_zlib': False,
                        'with_boost': False,
+                       'with_sse2': True,
```

Comment:
Shouldn't these options be removed on non-intel architectures?

Reply:
This options are opt-out (in botan itself) so default all options are enabled. So botan can handle it by themselves. 

I see no need for checking this.

Reply:
I mean I think it should add this in `config_options`:
```
if not self.settings.arch in ["x86", "x86_64"]:
    del self.options.with_sse2
    ...
```
arm, mips, risvc, ... do not have sse2

Reply:
I've added required checks